desktop/libs/indexer/src/data/solrconfigs/solrcloud/conf/schema.xml

```diff
@@ -55,15 +55,15 @@
 
     <fieldtype name="binary" class="solr.BinaryField"/>
 
-    <fieldType name="pint" class="solr.IntField"/>
+    <fieldType name="pint" class="solr.TrieIntField"/>
```

Comment:
I have tried to reverse the changes in the other files but could you help to solve or remove other files at the same time? Thanks